packages/react-components/src/templates/EventPage.tsx

```diff
@@ -57,6 +59,7 @@ const EventPage: React.FC<EventPageProps> = ({
             {formatDistance(new Date(), new Date(lastModifiedDate))} ago
           </small>
         </Paragraph>
+        {props.speakers.length && <SpeakerList {...props} />}
```

Comment:
```suggestion
        {!!props.speakers.length && <SpeakerList {...props} />}
```
JSX will print 0 without !!. Sorry